File: app/src/main/java/com/absoluteapps/arthurl/mobooru/HelpAdapter.java
package com.absoluteapps.arthurl.mobooru;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

public class HelpAdapter extends FragmentPagerAdapter {
    public HelpAdapter (FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        switch (position) {
            case 0:
                return HelpFragment.newInstance(position);
            case 1:
                return HelpFragment.newInstance(position);
            default:
                return HelpFragment.newInstance(position);
        }
    }

    @Override
    public int getCount() {
        return 3;
    }

}

File: app/src/main/java/com/absoluteapps/arthurl/mobooru/HelpAdapter.kt
package com.absoluteapps.arthurl.mobooru

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter

class HelpAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {

    override fun getItem(position: Int): Fragment {
        when (position) {
            0 -> return HelpFragment.newInstance(position)
            1 -> return HelpFragment.newInstance(position)
            else -> return HelpFragment.newInstance(position)
        }
    }

    override fun getCount(): Int {
        return 3
    }

}
